athena/modules/text/module_text_llm/tests/utils/mock_llm.py

```diff
@@ -0,0 +1,96 @@
+"""Mock language model implementations for testing the text LLM module."""
+
```

Comment:
Not sure if this is pythonic - to explain what a file is there for. I think the name is already clear enough 😊